DataLayer/Configuration.cs

```diff
@@ -11,7 +11,7 @@ public static class Configuration
         public static void ConfigureDbContext(IServiceCollection services, string connectionString) =>
             services.AddDbContext<GameBoardDbContext>(
                 options =>
-                    options.UseNpgsql(connectionString));
+                    options.UseSqlServer(connectionString));
```

Comment:
I'm relatively sure that migrations for Npgsql don't work with SQLServer. And even if, then for sure when you remove Npgsql packages the migration will not compile. You'll need to regenerate the initial Identity migration. On that note, I'd argue the default naming convention for tables in SQLServer is PascalCase as opposed to the Postgre snake_case, so it should be changed in the `DbContext.Model`.

Reply:
Done.